org.eclipse.scanning.api/src/org/eclipse/scanning/api/event/scan/DeviceValueMultiPosition.java

```diff
@@ -0,0 +1,82 @@
+/*-
+ * Copyright © 2017 Diamond Light Source Ltd.
+ *
+ * This file is part of GDA.
+ *
+ * GDA is free software: you can redistribute it and/or modify it under the
+ * terms of the GNU General Public License version 3 as published by the Free
+ * Software Foundation.
+ *
+ * GDA is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with GDA. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.eclipse.scanning.api.event.scan;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * This is a special Position class to allow both x and y gap values to be returned from a
+ * NexusSlitsWrapper scannable. It is a temporary creation until we understand why can't
+ * use a List or a Map for a DeviceValue in a DeviceRequest. 
+ *
+ */
+public class DeviceValueMultiPosition {
+	@JsonProperty("values")
+	private List<Double> values = new ArrayList<Double>() {{
+		add(0.0);
+		add(0.0);
+	}};
+
+	public double getX_gap() {
+		return values.get(0);
+	}
+
+	public void setX_gap(double x_gap) {
+		values.set(0, x_gap);
+	}
+
+	public double getY_gap() {
+		return values.get(1);
+	}
+
+	public void setY_gap(double y_gap) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00100)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00117)